homeassistant/components/wallbox/__init__.py

```diff
@@ -256,7 +258,7 @@ def device_info(self) -> DeviceInfo:
                     self.coordinator.data[CHARGER_DATA_KEY][CHARGER_SERIAL_NUMBER_KEY],
                 )
             },
-            name=f"Wallbox - {self.coordinator.data[CHARGER_NAME_KEY]}",
+            name=self.coordinator.data[CHARGER_NAME_KEY],
```

Comment:
why did you remove the Wallbox - prefix here? It makes the name more descriptive, now its only the type of the wallbox being displayed.

Reply:
Every entity would be `.wallbox_{name}_{entity}`, so I just removed it here.

Originally I skipped this integration since this had a device name mismatch (old entities were prefixed with the entry title and the device was called wallbox - something). So with this change the device name will be the prefix (so if you change the device name, it will directly change the rest as well).

I thought this was the most logical name, but like I said, I don't own one, so what would you prefer?

Reply:
currently the device (for me ) is called 'Wallbox - Copper SB{serialNumber}', the entities are called (for example) 'Wallbox Portal Charging Speed'. From a functional perspective I would consider removing the 'Portal' part in the entity title, apart from that the current names are very clear (my standpoint). 
![image](https://github.com/home-assistant/core/assets/10683625/e5ddb181-24b7-492a-a029-ae4578e26f8b)

Reply:
Do I understand correctly that the device name then should be just Wallbox? Keep in mind that this is just the default name and that you can edit the name for all entities in one go if you want to rename it.

Reply:
no, its quite common to have mulitple wallboxes, so including the type and serial in the name is useful (that's how it works right now).